### ISSUE TITLE
Stars in row

### ISSUE THREAD

**Issue by rysson** [matias-kovero/BetterUI#4]
Fist, thank you for awesome mod. It looks pretty.

Could you tell me how to get all stars in one row like in yours example:
![20210414-180758-133x84](https://user-images.githubusercontent.com/5898312/114742816-64d9c200-9d4c-11eb-800c-9989155a3718.png)

I have bigger stars:
![20210414-182409-194x94](https://user-images.githubusercontent.com/5898312/114745097-a10e2200-9d4e-11eb-9f40-bd8f66a4479a.png)

**Comment by matias-kovero** [comment]
Are you sure you are using the latest version, this was an issue in earlier versions 1.2.x and lower

**Comment by rysson** [comment]
Yes, I'm quite sure. I saw bars artifacts in 2.0.0 for example (removed in 2.0.1). Except, that 2.0.2 was not available yesterday :-)
```
.../Valheim/BepInEx/plugins$ md5sum $(find -type f -iname "*better*")
5d9b5019167313e4eab0d1e3533e8cb3  ./BetterUI/BetterUI.dll
```

And yes, I see this issue for long time, that's way I select numbers. But yours stars are looks awesome.
A removed all other mods and I have the same effect.

**Comment by matias-kovero** [comment]
At first glance it seems that the game has an different font spacing and its not taking my adjust to consider.

However the slot 7 has an really weird problem. Only 2 stars but both of them are on their own rows. In my code everything should be in 1 string and no row changes.

Sneak peak to code: [Items.cs#L96](https://github.com/matias-kovero/BetterUI/blob/7f579f4c1f6921440bf20cae3442dbaf19f02ff4/BetterUI/Patches/Items.cs#L96)

Could you provide:
System OS, System Language, Valheim Language.

**Comment by rysson** [comment]
Yes,. it's really strange. I see, you replace numer with stars character 2605 (nice trick), not with pixmaps. Than there is no way to put 2 stars in 2 rows, but it happens.

OS: Debian bullseye/sid (testing + unstable GNU / Linux)
GPU: GeForce RTX 2070 with nvidia drivers (from Debian repo), than Vulcan is used.

OS Language: it's a little harder, I use English, but same settings is from Polish (including rules like CTYPE):
```
$ locale
LANG=en_US.UTF-8
LANGUAGE=
LC_CTYPE=pl_PL.UTF-8
LC_NUMERIC="en_US.UTF-8"
LC_TIME=pl_PL.UTF-8
LC_COLLATE=pl_PL.UTF-8
LC_MONETARY=pl_PL.UTF-8
LC_MESSAGES="en_US.UTF-8"
LC_PAPER=pl_PL.UTF-8
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT=pl_PL.UTF-8
LC_IDENTIFICATION=pl_PL.UTF-8
LC_ALL=
```

Valheim Language: Polish.

I reset all `LC_*` vars and set English in Valheim – effect the same.
